app/src/main/java/com/shagalalab/sozlik/settings/LocaleHelper.java

```diff
@@ -0,0 +1,97 @@
+package com.shagalalab.sozlik.settings;
+
+import android.annotation.TargetApi;
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.content.res.Configuration;
+import android.content.res.Resources;
+import android.os.Build;
+import android.preference.PreferenceManager;
+
+import com.shagalalab.sozlik.R;
+
+import java.util.Locale;
+
+/**
+ * Created by QAREKEN on 4/8/2018.
+ */
+
+public final class LocaleHelper {
+    private static final String SELECTED_LANGUAGE = "Locale.Helper.Selected.Language";
+    private static LocaleHelper instance;
+    private LocaleHelper() {
+        //not called
+    }
+
+    public static LocaleHelper getInstance() {
+        if (instance == null) {
+            instance = new LocaleHelper();
+        }
+        return instance;
+    }
+
+    public static Context onAttach(Context context) {
+        String lang = getPersistedData(context, Locale.getDefault().getLanguage());
+        return setLocale(context, lang);
+    }
+
+    public static Context onAttach(Context context, String defaultLanguage) {
+        String lang = getPersistedData(context, defaultLanguage);
+        return setLocale(context, lang);
+    }
+
+    public static String getLanguage(Context context) {
+        return getPersistedData(context, Locale.getDefault().getLanguage());
```

Comment:
bul jerde `Locale.getDefault().getLanguage()` ornina `context.getString(R.string.pref_alphabet_default)` boliwi kerek, sonda eger locale'di taba almasa default'in qaytaradi